core/lib/Thelia/Core/Template/Loop/Image.php

```diff
@@ -361,11 +361,19 @@ public function parseResults(LoopResult $loopResult)
                 // Dispatch image processing event
                 $this->dispatcher->dispatch(TheliaEvents::IMAGE_PROCESS, $event);
 
+                $originalImageSize = getimagesize($sourceFilePath);
```

Comment:
You're sure ?
 $event->getOriginalFileUrl() return an url => http://mydomains.fr/cache/images/product/on_air_2-6.jpg
it seem that remote file are not the best choice for getImageSize()...